functions-methods.js

```diff
@@ -33,4 +54,24 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+console.log ("------------------ Opdracht 3 -------------------")
+
+function checkEmailValidity (emailAdress) {
+    const domainMonkey = emailAdress.indexOf("@");
+    const domainDot = emailAdress.lastIndexOf(".");
+    const domainComma = emailAdress.indexOf(",");
+
+    if ((domainMonkey >= 0) && (domainDot + 1 != emailAdress.length) && (domainComma < 0)){
+        console.log ("True")
+    }
+    else{
+        console.log ("False")
+    }
```

Comment:
Dit kun je nog wat versimpelen
```suggestion
    console.log(domainMonkey >= 0 && domainDot + 1 !== emailAdress.length && domainComma < 0);
```